### PR TITLE
Test Exit code of BFP test and ask for Expected difference

### PR DESCRIPTION
**Pull request by ja7**
On BFP differences the job stop and ask for a question in the Integration test stage. 

eksampel from  https://is.dbc.dk/job/vip/job/openagency-php/job/vp-62-bft-continue/1/console 

22:39:11  Input requested
22:39:28  Approved by Jacob Christian Poulsen

### CODE REVIEW

File: run-system-test.sh [hunk 1]
```diff
@@ -64,6 +71,7 @@ function main()  {
   if [[ ${RESULT} -ne 0 ]] ; then
     info "Dumping logs from ws image"
     getLast25ErrorLogLinesOfContainer "$WS_SERVICE"
+    info "ja7--------------------------------------------------------------------- ${DOCKER_COMPOSE} logs ${WS_SERVICE}"
```
Comment (thldbc) [inline run-system-test.sh:74]: Er det stadig meningen denne her loglinje skal være der?

File: Jenkinsfile [hunk 2]
```diff
@@ -82,10 +82,31 @@ pipeline {
...
-                                ./run-system-test.sh --debug --pull --tag ${DOCKER_BUILD_TAG}
-                                """
+                        def rc=sh script: "./run-system-test.sh --debug --pull --tag ${DOCKER_BUILD_TAG}", returnStatus: true
+                        junit '**/TEST-junit-jupiter.xml'
+                        
+                        /*def rc=sh(script: """#!/bin/bash
+                                              exit 123
+                                              """,
+                                returnStatus: true)
+                        */
```
Comment (mran-dbc) [inline Jenkinsfile:92]: Slettes?

File: run-system-test.sh [hunk 3]
```diff
@@ -32,15 +32,22 @@ function runTest() {
   info "Starting oa brute force tester"
   TEST_IP_PORT=$(getIPAndPortOfContainer "$WS_SERVICE")
   GOLD_IP_PORT=$(getIPAndPortOfContainer "$WS_SERVICE_GOLD")
-  docker run --rm -e BUILD_NUMBER -v "$JUNIT_RESULT_DIR:/output" docker-i.dbc.dk/oa-tester "$brute_force_mode" "http://$GOLD_IP_PORT/gold_oa/" "http://$TEST_IP_PORT/test_oa/"
+  set +e # manually handle exit code from os-tester
+  docker run --rm -e BUILD_NUMBER -v "$JUNIT_RESULT_DIR:/output" docker-i.dbc.dk/oa-tester "$brute_force_mode" "http://$GOLD_IP_PORT/gold_oa/" "http://$TEST_IP_PORT/test_oa/" FindLibraryTest
```
Comment (mran-dbc) [inline run-system-test.sh:36]: FindLibraryTest skal vist ikke være der

File: run-system-test.sh (hunk 1 above)
Comment (ja7) [reply run-system-test.sh:74]: ups
godt fanget.

File: Jenkinsfile (hunk 2 above)
Comment (ja7) [reply Jenkinsfile:92]: Man skal husk at få push'et alle sine commits.

**Pull request closed without merging**